h2o-py/h2o/model/PermutationFeatureImportance.py

```diff
@@ -0,0 +1,23 @@
+# -*- encoding: utf-8 -*-
+"""
+H2O Model Reliance.
+
+:copyright: (c) 2020 H2O.ai
+:license:   Apache License Version 2.0 (see LICENSE for details)
+"""
+
+from h2o.frame import H2OFrame
+from h2o.expr import ExprNode
+
+
+def permutation_featue_importance(frame, model):
+    
+    if type(frame) is H2OFrame:
+        m_frame = H2OFrame._expr(ExprNode("Perm_Feature_importance", frame, model.model_id))
+    else:
+        raise ValueError("Frame is not H2OFrame")
+    
+    # TODO 
```

Comment:
If I expect correct, this return H2OFrame because you are returning it in your Ast. You can inspire how to deal with this in https://github.com/h2oai/h2o-3/blob/9c86f1f64bf25027490d4781f5e16c8ba987e96e/h2o-py/h2o/model/model_base.py#L476. You can use frame here, you cannot use TwoDimTable here. Try to debug how the structure from Ast looks; it is always some JSON which is formatted to predefined Python or R structures.